build/8-to-9-migration/migrate.sh

```diff
@@ -23,6 +23,9 @@ ${mvn_cmd} rewrite:run \
 find "${script_dir_path}/../../optaplanner-spring-integration" -type f -name "spring.factories" -exec rm {} \;
 
 if [[ ! "$1" == "test" ]]; then
+  # The formatter and impsort goals override validation activated by the CI environment variable.
+  ${mvn_cmd} process-test-sources -Dformatter.goal=format -Dimpsort.goal=sort -Denforcer.skip
```

Comment:
The formatting has to go first, as after incrementing the major version, the build may fail due to missing new artifacts.